active_previous_actived_cron/models/ir_cron.py

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+# © 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class IrCron(models.Model):
+    _inherit = 'ir.cron'
+
+    active_old = fields.Boolean(
+        string='Intial active state',
+        help="State of the cron before 'Active Previous Active Cron' "
+             "module installation.")
+
+    def _inactive_crons(self):
+        crons = self.search([])
+        if crons:
```

Comment:
I don't think this check is necessary, a `write` on an empty recordset is basically a noop